### PR TITLE
Fix modifiers are not in the canonical preferred order

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Fix modifiers are not in the canonical preferred order (as stated in the Java Language Specification).

### CODE REVIEW

File: src/main/java/org/isf/dicom/manager/DicomManagerFactory.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public static Long getMaxDicomSizeLong() throws OHDicomException {
 	 * return the manager for DICOM acquired files
 	 * @throws OHDicomException 
 	 */
-	public synchronized static DicomManagerInterface getManager() throws OHDicomException {
+    public static synchronized DicomManagerInterface getManager() throws OHDicomException {
```
Comment (mwithi) [inline src/main/java/org/isf/dicom/manager/DicomManagerFactory.java:51]: could please you use the https://github.com/informatici/openhospital-core/blob/develop/.editorconfig included in the project?

File: src/main/java/org/isf/dicom/manager/SourceFiles.java [hunk 2]
```diff
@@ -209,8 +209,8 @@ else if (isDicom) {
 	 * @param patient
 	 * @throws Exception
 	 */
-	@SuppressWarnings("unused")
-	public synchronized static void loadDicom(FileDicom dicomFileDetail, File sourceFile, int patient) throws Exception {
+    @SuppressWarnings("unused")
+    public static synchronized void loadDicom(FileDicom dicomFileDetail, File sourceFile, int patient) throws Exception {
```
Comment (mwithi) [inline src/main/java/org/isf/dicom/manager/SourceFiles.java:213]: could please you use the https://github.com/informatici/openhospital-core/blob/develop/.editorconfig included in the project?

File: src/main/java/org/isf/dicom/manager/DicomManagerFactory.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/dicom/manager/DicomManagerFactory.java]: As noted in the other PR I am using it but it appears that either the files have some mix of indentation when I started or the default tab width is not set or defined.

File: src/main/java/org/isf/dicom/manager/SourceFiles.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/dicom/manager/SourceFiles.java]: See above.

**Comment by dbmalkovsky** [comment]
I am seriously thinking of closing this PR and trying it again fresh.

**Comment by dbmalkovsky** [comment]
Closing this PR and will reopen a new one with the 7 files formatted to match the recent IDE code style.

**Pull request closed without merging**